.github/actions/check-links/action.yml

```diff
@@ -4,7 +4,7 @@ runs:
   using: "composite"
   steps:
     - name: install-releaser
-      uses: jupyter-server/jupyter_releaser/.github/actions/install-releaser@v1
+      uses: davidbrochart/jupyter_releaser/.github/actions/install-releaser@fix_multipy
```

Comment:
We'll need to revert that one (and similar below) to point to the releaser actions.